spec/support/ovirt_sdk/connection_vcr.rb

```diff
@@ -33,6 +33,7 @@ def wait(request)
       end
       res = super(request)
       @all_req_hash[req_key] << http_response_to_hash(res)
+      FileUtils.mkdir_p(File.dirname(path_to_recording))
```

Comment:
This makes the behavior closer to VCR by creating the directory structure leading up to the file location.